docs/admin/workloads/workload-overview-admin.md

```diff
@@ -2,25 +2,54 @@
 
 ## Workloads
 
-Run:ai schedules __Workloads__. Run:ai workloads are comprised of:
+Run:ai schedules *Workloads*. Run:ai workloads are comprised of:
 
-* The _Kubernetes object_ (Job, Deployment, etc) which is used to launch the container, inside which the data science code runs. 
-* A set of additional resources that are required to run the Workload. Examples: a service entry point that allows access to the Job, a persistent volume claim to access data on the network, and more. 
+* The *Kubernetes object* (Job, Deployment, etc) which is used to launch the container, inside which the data science code runs.
+* A set of additional resources that are required to run the Workload. Examples: a service entry point that allows access to the Job, a persistent volume claim to access data on the network, and more.
 
-All of these components are created together and deleted together when the Workload ends. 
+All of these components are created together and deleted together when the Workload ends.
 
 Run:ai currently supports the following Workloads types:
 
 |  Workload Type | Kubernetes Name | Description |
 |----------------|-----------------|-------------|
 | Interactive    | `InteractiveWorkload` | Submit an interactive workload |
 | Training       | `TrainingWorkload`| Submit a training workload |
-| Distributed Training | `DistributedWorkload` | Submit a distributed training workload using TensorFlow, PyTorch or MPI | 
+| Distributed Training | `DistributedWorkload` | Submit a distributed training workload using TensorFlow, PyTorch or MPI |
 | Inference      | `InferenceWorkload` | Submit an inference workload |
 
+## Workloads View
+
+The *Workloads* view provides a more advanced UI than the previous *Jobs* UI.
+
+To enable the *Workloads* view, go to the *Jobs* table and toggle the switch from `Jobs` to `Workloads`. To return, switch the toggle from `Workloads` to `Jobs`.
+
+!!! Note
+    To submit a workload, use either the *Jobs* view or the CLI.
```

Comment:
There will be an option to submit a Workspace or a Trainings workload from the new view.

Reply:
I am not seeing it in 2.15 - when this changes I will add it.

```suggestion
    To submit a workload, use either the *Jobs* view or the CLI.
    
    <!-- remove this when the feature is added -->
```